src/components/new-request/search.vue

```diff
@@ -342,6 +295,24 @@ export default class NewSearch extends Vue {
   }
 
   // Local Getters
+  get isPremium () {
+    return JSON.parse(sessionStorage.getItem('CURRENT_ACCOUNT'))?.accountType === 'PREMIUM'
```

Comment:
Would be great to have a const or enum for account type.

Reply:
sure thing will update